src/components/VideoList/VideoItem/VideoItem.jsx

```diff
@@ -25,17 +26,19 @@ const VideoItemVerticalStyled = styled.li`
   flex-direction: row;
   height: 98px;
   padding: 6px 10px;
-  border: 1px solid #eee;
+  border: ${({ theme }) => theme.videoItemBorder};
 `;
 
 const VideoItem = (props) => {
+  const { themes, currentTheme } = useTheme();
+
   return props.display === 'vertical' ? (
-    <VideoItemVerticalStyled>
+    <VideoItemVerticalStyled theme={themes[currentTheme]}>
```

Comment:
Just as a reference styled components has a way to define Themes ([you can check this ref](https://styled-components.com/docs/tooling#styled-theming)), but this is a good exercise to try to implement it by yourself no need to change the code 👍